__tests__/server/server.js

```diff
@@ -45,15 +45,23 @@ describe('Api test', () => {
   let server;
   beforeEach(() => {
     process.env.NODE_ENV = 'test';
+    process.env.SERVER_API_KEY = '79b2d5eb-c1fd-42c4-9391-6b2c9780d591';
     jest.resetModules();
     server = require(MODULE).default;
   });
   test('post to /community-app-assets/api/logger', () => request(server).post('/community-app-assets/api/logger')
+    .set('Authorization', `ApiKey ${process.env.SERVER_API_KEY}`)
```

Comment:
everywhere you read API key as `process.env.SERVER_API_KEY`, you should actually get it via Node config, i.e.
```
import config from 'utils/config'; // Cause Node config is server-side only, and our auxiliary utils/config modules hides the difference between client- and server-side configs.
...
.set('Authorization', `ApiKey ${config.SERVER_API_KEY}`)
```